torchrl/envs/gym_like.py

```diff
@@ -144,7 +144,7 @@ def read_done(self, done):
             done (np.ndarray, boolean or other format): done state obtained from the environment
 
         """
-        return done, done
+        return done, done.any() if not isinstance(done, bool) else done
```

Comment:
I don't particularily like it 
One aspect to consider is when we're doing our skip_frame but one of the envs is done and not the others. Then we stop the loop. This means that a bunch of envs silently don't complete the loop. Not sure what to do bc we can't really keep on applying the same action to the other env if it's in auto-reset.

One solution is that if a non-complete done is encountered we raise a warning telling users to use the skip-frame transform instead.

Reply:
I think the situation you described is too common and silently not completing the loop is dangerous.

A bit hacky but can we create some kind on NO-OP step wrapper (e.g. if action is -1, returns the last obs, reward etc)? and overwrite the envs that have reached.

You do the loop skip_frame time but for early finishing envs a useless step will be called at the end.

Reply:
What about disabling frame_skip for num_envs > 0, and just say that one needs to use the transform instead?

Reply:
Oh I always thought frame_skip as a transform was more consistent with the rest of the library. I thought there was some problem that made it less efficient or sth. But yes, it that is not the case, I think using the transform is the best way.

Reply:
@vmoens quick heads up here. I think the frame_skip transform did not handle truncation/termination correctly the last time I checked. Also, some envs natively handle frame_skip and this runs much faster as they don't have to render the frames, e.g. ALE-v5 which skips the frames internally with the `frameskip` keyword in the constructor.